internal/metamorph/store/postgresql/postgres.go

```diff
@@ -464,12 +464,38 @@ func (p *PostgreSQL) UpdateStatusBulk(ctx context.Context, updates []store.Updat
 		;
     `
 
-	rows, err := p.db.QueryContext(ctx, qBulk, pq.Array(txHashes), pq.Array(statuses), pq.Array(rejectReasons), p.hostname, metamorph_api.Status_SEEN_ON_NETWORK, metamorph_api.Status_MINED)
+	tx, err := p.db.Begin()
 	if err != nil {
+		panic(err)
```

Comment:
panic stops the whole program from execution. I think we should simply return an error here.

Reply:
I left it because the error is so unexpected here and also if there is error it means the application cannot even run anymore.Db is locked so IDK, doesn't change much removing this

Reply:
I think it is better to return the error as usual, instead of shutting down the whole service in an uncontrolled way using `panic`. IMO there should be no panics anywhere, except maybe in examples.

If the goal is to shutdown the service completely, then waiting for the particular error and shutting down the service gracefully would be the better option.